src/frontend/components/SøknadsSteg/Forside/Forside.tsx

```diff
@@ -15,7 +15,7 @@ import InnholdContainer from '../../Felleskomponenter/InnholdContainer/InnholdCo
 import TekstBlock from '../../Felleskomponenter/TekstBlock';
 
 import BekreftelseOgStartSoknad from './BekreftelseOgStartSoknad';
-import FortsettPåSøknad from './FortsettPåSøknad';
+import { FortsettPåSøknad } from './FortsettPåSøknad';
```

Comment:
Bra at du fikset `FortsettPåSøknad` til å bli en named export i stedet for en default export. 🎉